src/components/media-loader.js

```diff
@@ -48,14 +62,26 @@ AFRAME.registerComponent("media-loader", {
   },
 
   showLoader() {
-    const loadingObj = new THREE.Mesh(new THREE.BoxGeometry(), new THREE.MeshBasicMaterial());
+    const useFancyLoader = !!loadingObject;
```

Comment:
Was gpong to suggest just making a promise to loadingObject instead of having these 2 code paths but can't decide if that actually ends up being cleaner.